source/account_structure/index.html.md.erb

```diff
@@ -1,92 +1,68 @@
+- give each service its own name on the GOV.UK Pay payment pages
+- report on each service's payments separately
+- send each service's payments to different bank accounts
+- give your staff members access to each service separately in the GOV.UK Pay admin tool
 
-The diagram show two services, ‘Parking permits’ and ‘Parking fines’, that are considered separate services by
-both GOV.UK Pay and the PSP.
+To set up your services like this, [create a new GOV.UK Pay service](https://selfservice.payments.service.gov.uk/my-services/create) for each of your online services.
 
-If you use this account structure, you can:
+GOV.UK Pay services are free, and there's no limit to how many you can create.
 
-- give access for each service to different staff members
-- optionally use different service names on the GOV.UK Pay payment pages
-- send each service's payments to a different bank account
-- report separately on each of your services
+To change a service's name, go to the [**My services** page in the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services).
 
-### Type 2 (GOV.UK Pay combined, PSP combined)
+### Send your users to a combined GOV.UK Pay service
 
-<%= image_tag "/images/accountstructure_2.svg", { :alt => '' } %>
+Send your users to a combined GOV.UK Pay service if you need to:
 
-The diagram shows two services, ‘Parking permits’ and ‘Parking fines’, that are considered a single combined
-service by GOV.UK Pay and the PSP.
+- report on all the services' payments together
+- change settings easily across all the services, for example which card types you accept
 
-You may want to use this configuration if you want to simplify the administration of multiple services in GOV.UK Pay. For example, if you do not need reporting at an individual service level.
+You will not be able to give the services different names or send payments to different bank accounts.
 
-The transactions from both services will appear in the same transactions list and CSV file. You may find it useful to use reference numbers to separate payments clearly, for example “parking_fine_1234” and “parking_permit_1234”.
+You may need to use a different [reference number](/making_payments/#reference) or add [custom metadata](/custom_metadata/#add-custom-metadata) for each of your services, so you can tell which service each payment comes from.
 
-In this configuration, payments to services must go to the same bank account because
-they are a single service in the PSP. It is not possible to support multiple PSP accounts in a single GOV.UK Pay service.
+To set up a group of your services like this, use the same API key or payment link for all the services in the group.
 
-### Type 3 (GOV.UK Pay separate, PSP combined)
+## Decide where to send payments
 
-<%= image_tag "/images/accountstructure_3.svg", { :alt => '' } %>
+To help you reconcile payments, each GOV.UK Pay service can send payments to a:
```

Comment:
I think we can make this a bit more specific about the options with Stripe and Worldpay? (i'm relatively happy to ignore epdq and smartpay since we're not onboarding new people, so current users will likely have figured this out by now)

If you use Stripe, you will get a separate payout for each service in Pay. You can choose to have all payouts go into the same bank account, or provide a different bank account for each one during the go live process. You can set the description that will appear on your bank statement next to each payout.

If you use Worldpay, you add a Merchant code to each service. You will receive one payout for each merchant code you use. You can choose to add the same Merchant code to all Pay services, or you can use different Merchant codes. (I don't think Worldpay charge for merchant codes under the gov banking contract)

Reply:
Thanks for this - added!